init.sql

```diff
@@ -0,0 +1,8 @@
+GRANT ALL PRIVILEGES ON app_db.* TO 'app_user'@'%';
```

Comment:
Granting ALL PRIVILEGES to 'app_user' can be a security risk. Consider granting only the necessary permissions that the user needs to perform their tasks, such as SELECT, INSERT, UPDATE, and DELETE.

Reply:
Your job is to check the correctness of the implementation, not to provide additional tasks or conditions.